libraries/api/discussion_helper.cpp

```diff
@@ -196,15 +222,56 @@ namespace golos { namespace api {
             auto vshares = db.calculate_vshares(d.net_rshares.value > 0 ? d.net_rshares.value : 0);
 
             u256 r2 = to256(vshares); //to256(abs_net_rshares);
-            r2 *= pot.amount.value;
+            r2 = (r2 * d.reward_weight) / STEEMIT_100_PERCENT;
+            //r2 *= pot.amount.value;
+            r2 *= 16152;
```

Comment:
16152 - what does this magic number mean?

Reply:
@afalaleev For the testing purposes. Just don't removed because it is not a last commit.